src/Misc/expressionFunc/hashFiles/LICENSE

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
did you mean to checkin this file here?

Reply:
it's copied from the actions/typescript-action-template...
i will remove this